cobbler/cobbler_collections/collection.py

```diff
@@ -488,6 +487,58 @@ def _deserialize(self) -> None:
         self.inmemory = True
         self.deserialize_running = False
 
+    def add_to_indexes(self, ref: "ITEM") -> None:
+        """
+        Add indexes for the object.
+        :param ref: The reference to the object whose indexes are updated.
+        """
+        indx_dict = self.indexes["uid"]
+        indx_dict[ref.uid] = ref.name
+
+    def remove_from_indexes(self, ref: "ITEM") -> None:
+        """
+        Remove index keys for the object.
+        :param ref: The reference to the object whose index keys are removed.
+        """
+        indx_dict = self.indexes["uid"]
+        indx_dict.pop(ref.uid, None)
+
+    def find_by_indexes(self, kargs: Dict[str, Any]) -> Optional[List["ITEM"]]:
```

Comment:
Could we rename `kargs` to `kwargs`? In my mind, `kargs` is a bit confusing to me, it's either `args`, or `kwargs`.

I see btw that the codebase uses a lot of `kargs`, which seem interchangeable with `kwargs`. Confusingly, it also uses `kwargs`.

Reply:
Agreed. I'll open an issue for it.

Reply:
https://github.com/cobbler/cobbler/issues/3732